docs/indepth/installation.rst

```diff
@@ -347,10 +347,37 @@ You must also install the ZFS package repository.
 The following commands will install the two repositories and the ``clusterhq-flocker-node`` package.
 Paste them into a root console on the target node:
 
-.. task:: install_flocker
+.. task:: install_flocker fedora-20
    :prompt: [root@node]#
 
-Installing ``clusterhq-flocker-node`` will automatically install Docker, but the ``docker`` service may not have been enabled or started.
+Installing on CentOS 7
+----------------------
+
+Flocker requires the latest available kernel.
+
+.. task:: upgrade_kernel_centos
+
+Flocker requires ZFS, and installing ZFS requires that the running kernel be the one that will eventually be used.
+Thus we need to reboot into the new kernel.
+
+.. prompt:: bash [root@node]#
+
+   shutdown -r now
```

Comment:
@wallrj showed me that in the automation we use libcloud reboot API - because, we think, fabric gets upset if you use it to reboot the machine.  That seems like a reasonable technical justification for doing things this way for now, so if that's the reason here then nothing needs to change.